truss/cli/cli.py

```diff
@@ -703,7 +703,7 @@ def push(
     #  config checks.
     if tr.spec.config.secrets and not trusted:
```

Comment:
While we're at it, can we move this up to before the zip and push operation?

Reply:
sure. Honest feedback here -- should this block the deploy period? Why just warn?

Reply:
I think warn is fine as long as its early enough to do something about it. Right now, its too late to do anything about it